src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -305,11 +305,25 @@ Below is a list of available sub-commands:
 <!-- markdownlint-disable line-length -->
 | Flag                             | Description
 |:---------------------------------|:----------------------------------------------------------------------------|
-| `--namespace` `<string>`         | Namespace in which validation pods should be deployed. If not specified, the `default` namespace is used
+| `--namespace` `<string>`         | Namespace in which validation pods should be deployed. If not specified, the `submariner-operator` namespace is used
 | `--validation-timeout` `<value>` | Timeout in seconds while validating the connection attempt
 | `--verbose`                      | Produce verbose logs during validation
 <!-- markdownlint-enable line-length -->
 
+{{% notice warning %}}
+Starting with Kubernetes 1.23, the Pod Security admission controller expects namespaces to have security labels. Without these, you will see
+warnings in `subctl`'s output. This will not actually impact `subctl`, but you can avoid the warnings and ensure correct behavior by adding
+these labels to the user specified `namespace`:
```

Comment:
```suggestion
Starting with Kubernetes 1.23, the Pod Security admission controller expects namespaces to have certain security labels.
Without these labels, when using the `--namespace` flag you might see warnings in `subctl`'s output (this has no detrimental impact on `subctl diagnose`).
In order to avoid the warnings and ensure correct behavior by adding these labels to the namespace:
```

Reply:
> In order to avoid the warnings and ensure correct behavior by adding these labels to the namespace:

Suggestion-suggestion:

> Avoid the warnings and ensure correct behavior by adding these labels to the namespace: